gnat2goto/driver/tree_walk.adb

```diff
@@ -1483,12 +1632,24 @@ package body Tree_Walk is
          declare
             Op_Geq : constant Irep := New_Irep (I_Op_Geq);
             Op_Leq : constant Irep := New_Irep (I_Op_Leq);
+            Lower_Bound : constant Irep := New_Irep (I_Constant_Expr);
+            Upper_Bound : constant Irep := New_Irep (I_Constant_Expr);
          begin
             Set_Lhs (Op_Geq, I);
-            Set_Rhs (Op_Geq, Get_Lower_Bound (Sym_Type));
+            Set_Type (I     => Lower_Bound,
+                      Value => Sym_Type);
+            Set_Value (I     => Lower_Bound,
+                       Value => Load_Bound_In_Hex (Get_Lower_Bound (Sym_Type),
+                        Pos (Get_Width (Sym_Type))));
```

Comment:
What if the width of the type isn't large enough to hold the value?

Reply:
Wouldn't that mean that the bound cannot be stored in a variable of that type?